src/pages/DrumMachinePage.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styled from 'styled-components';
+import * as Tone from 'tone';
+import DrumPad from '../components/DrumPad';
+
+export default function DrumMachine() {
+  return (
+    <PadList>
+      <DrumPad
+        handleClick={handleClick}
+        value={'../audio/Samples/Scratch1.wav'}
+      />
+      <DrumPad handleClick={handleClick} value={'./audio/Samples/Horn1.mp3'} />
+      <DrumPad
+        handleClick={handleClick}
+        value={'./audio/Samples/SynthShot1.wav'}
+      />
+      <DrumPad
+        handleClick={handleClick}
+        value="./audio/Samples/CongaFinger.wav"
+      />
```

Comment:
Maybe you can work with the map method to create the individual drum pads.
Maybe it is possible to store the values in an array and then map over them.

Reply:
I've already thought about that, but I think I'll do that later when I can change the samples on the buttons.